### PR TITLE
Added 301 redirects

### PR DESCRIPTION
**Pull request by cwarnermm**

### CODE REVIEW

File: source/conf.py [hunk 1]
```diff
@@ -199,6 +199,31 @@ def setup(app):
...
+    "administration/config-settings.html#google-settings": 
+        "https://docs.mattermost.com/configure/configuration-settings.html#google-settings",
+    "administration/config-settings.html#office-365-settings": 
+        "https://docs.mattermost.com/configure/configuration-settings.html#office-365-settings",
+    "administration/config-settings.html#openid-connect-other-settings": 
+        "https://docs.mattermost.com/configure/configuration-settings.html#openid-connect-other-settings",
+    "administration/config-settings.html#storage": 
+        "https://docs.mattermost.com/configure/configuration-settings.html#local-storage-directory",
+    "administration/config-settings.html#aggregate-search-indexes": 
+        "https://docs.mattermost.com/configure/configuration-settings.html#aggregate-search-indexes"
```
Comment (cwarnermm) [inline source/conf.py:214]:
```suggestion
        "https://docs.mattermost.com/configure/configuration-settings.html#aggregate-search-indexes":
```

File: source/conf.py [hunk 2]
```diff
@@ -199,6 +199,31 @@ def setup(app):
...
+    "administration/config-settings.html#enable-document-search-by-content": 
+        "https://docs.mattermost.com/configure/configuration-settings.html#enable-document-search-by-content",
+    "administration/config-settings.html#site-url": 
+        "https://docs.mattermost.com/configure/configuration-settings.html#site-url",
+    "administration/config-settings.html#enable-link-previews": 
+        "https://docs.mattermost.com/configure/configuration-settings.html#enable-link-previews",
+    "administration/config-settings.html#enable-high-availability-mode": 
+        "https://docs.mattermost.com/configure/configuration-settings.html#enable-high-availability-mode",
+    "administration/config-settings.html#forward-port-80-to-443": 
+        "https://docs.mattermost.com/configure/configuration-settings.html#forward-port-80-to-443"
```
Comment (cwarnermm) [inline source/conf.py:224]:
```suggestion
        "https://docs.mattermost.com/configure/configuration-settings.html#forward-port-80-to-443":
```

File: source/conf.py [hunk 3]
```diff
@@ -199,6 +199,31 @@ def setup(app):
...
+    "administration/config-settings.html#google-settings": 
+        "https://docs.mattermost.com/configure/configuration-settings.html#google-settings",
+    "administration/config-settings.html#office-365-settings": 
+        "https://docs.mattermost.com/configure/configuration-settings.html#office-365-settings",
+    "administration/config-settings.html#openid-connect-other-settings": 
+        "https://docs.mattermost.com/configure/configuration-settings.html#openid-connect-other-settings",
+    "administration/config-settings.html#storage": 
+        "https://docs.mattermost.com/configure/configuration-settings.html#local-storage-directory",
+    "administration/config-settings.html#aggregate-search-indexes": 
+        "https://docs.mattermost.com/configure/configuration-settings.html#aggregate-search-indexes":
```
Comment (cwarnermm) [inline source/conf.py:214]:
```suggestion
        "https://docs.mattermost.com/configure/configuration-settings.html#aggregate-search-indexes",
```

File: source/conf.py [hunk 4]
```diff
@@ -199,6 +199,31 @@ def setup(app):
...
+    "administration/config-settings.html#enable-document-search-by-content": 
+        "https://docs.mattermost.com/configure/configuration-settings.html#enable-document-search-by-content",
+    "administration/config-settings.html#site-url": 
+        "https://docs.mattermost.com/configure/configuration-settings.html#site-url",
+    "administration/config-settings.html#enable-link-previews": 
+        "https://docs.mattermost.com/configure/configuration-settings.html#enable-link-previews",
+    "administration/config-settings.html#enable-high-availability-mode": 
+        "https://docs.mattermost.com/configure/configuration-settings.html#enable-high-availability-mode",
+    "administration/config-settings.html#forward-port-80-to-443": 
+        "https://docs.mattermost.com/configure/configuration-settings.html#forward-port-80-to-443":
```
Comment (cwarnermm) [inline source/conf.py:224]:
```suggestion
        "https://docs.mattermost.com/configure/configuration-settings.html#forward-port-80-to-443",
```

File: source/conf.py [hunk 5]
```diff
@@ -199,6 +199,31 @@ def setup(app):
...
+    "administration/config-settings.html#smtp-email-server": 
+        "https://docs.mattermost.com/configure/configuration-settings.html#smtp-email-server"
```
Comment (cwarnermm) [inline source/conf.py:226]:
```suggestion
        "https://docs.mattermost.com/configure/configuration-settings.html#smtp-email-server",
```

**Review by justinegeffen — APPROVED** [review]

File: source/conf.py [hunk 6]
```diff
@@ -199,6 +199,31 @@ def setup(app):
     "administration/data-retention": "https://docs.mattermost.com/comply/data-retention-policy.html",
     "administration/custom-terms-of-service": "https://docs.mattermost.com/comply/custom-terms-of-service.html",
     "administration/mobile-changelog": "https://docs.mattermost.com/deploy/mobile-app-changelog.html",
+    "administration/config-settings.html#push-notification-contents": "https://docs.mattermost.com/configure/configuration-settings.html#push-notification-contents",
```
Comment (justinegeffen) [inline source/conf.py:202]: Non-blocking, but should the redirect URL be on a new line?

Comment (cwarnermm) [reply source/conf.py:202]: Good question! We can split the redirects up for readability, but it's not required.

**Pull request merged**